iris_grib/_save_rules.py

```diff
@@ -953,9 +953,9 @@ def set_fixed_surfaces(cube, grib, full3d_cube=None):
                    'maximum value = {}.')
             raise ValueError(msg.format(max_valid_nlevels, n_levels))
         # In sample data we have seen, there seems to be an extra missing data
-        # value after each set of n-levels coefficients.
-        # For now, we retain that encoding. This should not cause problems as
-        # values are indexed according to model-level-number,
+        # value *before* each set of n-levels coefficients.
+        # Note: values are indexed according to model_level_number, using
+        # zero-based numbering,
```

Comment:
I think the 'using zero-based numbering' bit is still confusing!
You might add to the following ..
`# I.E. sigma, delta = PV[i], PV[NV/2+i]  : where i=1..n_levels`